test/e2e/integreatly_test.go

```diff
@@ -114,6 +113,11 @@ func TestIntegreatly(t *testing.T) {
 		}
 	})
 
+	t.Fatal("Failed test")
```

Comment:
I assume this is here for testing purposes?

Reply:
yes, that's right